### DISCUSSION TITLE
Creation of dynamic property GraphQL\Type\Definition\InputObjectField::$rules is deprecated

### DISCUSSION THREAD

**Discussion by dikirill** [webonyx/graphql-php#1404] [General]
Hello
We are trying to update to PHP8.2 and now see these messages coming from your library. We can not go to version 15 now, we have to stick to 14 for now.
Can you please add `#[AllowDynamicProperties]` to these classes?

Thank you

**Comment by spawnia** [comment]
Older versions are generally no longer supported, although exceptions may be made for sponsors.

**Comment by dikirill** [comment]
Donated. I hope this is enough :)

**Reply by spawnia** [reply]
Thank you, fixed with https://github.com/webonyx/graphql-php/releases/tag/v14.11.10.

**Comment by dikirill** [comment]
Thank you very much @spawnia !!
